website_livechat_firstname/static/description/icon.svg

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This file can be removed. The SVG isn't necessary unless you're providing a custom icon. It's used in the event someone wants to make changes down the road - we have this saved in the template though.